app/events/decidim/challenges/publish_challenge_event.rb

```diff
@@ -0,0 +1,23 @@
+# frozen-string_literal: true
+
+module Decidim
+  module Challenges
+    class PublishChallengeEvent < Decidim::Events::SimpleEvent
```

Comment:
Is this event being sent somewhere? shouldn't we remove it and the corresponding translations?